debian/source/lintian-overrides

```diff
@@ -11,8 +11,11 @@ version-substvar-for-external-package mariadb-client-10.6 -> mysql-client-core-8
 version-substvar-for-external-package libmariadbd-dev -> libmariadbclient-dev
 # ColumnStore not used in Debian, safe to ignore. Reported upstream in https://jira.mariadb.org/browse/MDEV-24124
 source-is-missing storage/columnstore/columnstore/utils/jemalloc/libjemalloc.so.2
-# Must be fixed upstream
+# Should be fixed upstream
 source-is-missing storage/mroonga/vendor/groonga/examples/dictionary/html/js/jquery-ui-*.custom.js
+# Should be fixesd in upstream Source code is missing in new Lintian style
```

Comment:
Typo. `fixesd` -> `fixed`.

Should it not be two sentences? The comment doesn't make sense to me otherwise.
```
# Should be fixed in upstream. Source code is missing in new Lintian style
```

Aslo, what exactly does this mean, "missing in new Lintian style" ? Isn't there a better phrasing possible for this?

Reply:
Very heavy and really bad cut'n'paste and changed ordering at the end and worded little bit differently.